### DISCUSSION TITLE
Using Sktime transformer pipelines in Skforecast

### DISCUSSION THREAD

**Discussion by vpekar** [JoaquinAmatRodrigo/skforecast#860] [General]
Hi,

I've made a fork of Skforecast that makes it possible to include Sktime transformer pipelines (e.g., deseasonaliser, detrender, boxcox,etc) into the Skforecast workflow. An example notebook can be found here: https://gist.github.com/vpekar/51921bb3e6be75afd2c8666a9f1ff250. The fork is here: https://github.com/vpekar/skforecast

If there is any interest, I will create a pull request.

**Comment by JavierEscobarOrtiz** [comment]
Hi @vpekar,

Thanks for reaching out and creating the examples! We will be evaluating this feature for the next release. Do you have any suggestions that we can consider?

We will keep you updated.

Javier

**Comment by JoaquinAmatRodrigo** [comment]
Of course! We are always open to new contributions. 
Could you mention the main changes you had to make to enable the use of SKtime transformers in SkForecast?
This will help us a lot to evaluate possible side effects of the changes.

**Reply by vpekar** [reply]
Hi both,

I've created a pull request and described main changes in the request. All unittests pass :) 

These changes are implemented only for Bayesian search at the moment. Later on, I can add them also for the exhaustive and random grid search.

Viktor

**Reply by JoaquinAmatRodrigo** [reply]
Thanks a lot @vpekar !

We will review it in the comming weeks ;)
